Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/UserDAO.java

```diff
@@ -127,4 +127,13 @@ public Long count(String query) throws DAOException {
     public void refresh(User user) {
         refreshObject(user);
     }
+
+    /**
+     * Get all active users sorted by surname and name.
+     *
+     * @return sorted list of all active users as User objects
+     */
+    public List<User> getAllActiveUsersSortedByNameAndSurname() {
+        return search("FROM User WHERE active = 1 AND visible IS NULL ORDER BY surname ASC, name ASC");
```

Comment:
Is `visible` not changed to an integer value and renamed to  `deleted` and `NULL` should not possible any more?

Reply:
true